_includes/working-with-tables.md

```diff
@@ -0,0 +1,208 @@
+1. In the _Concentration_ section, select **<=** from the dropdown menu and enter **3** into the input field.
+1. Click **Check**.
+    * Note that the _QC Passed?_ values have changed.
+
+<a name="export"/>
+
+### Export
+The current state of the bulk table data can be exported at any time. Exports can be saved as Microsoft Excel, Open Document Format, or Comma-delimited Data (CSV) files.
+
+1. Return to your _Create Libraries_ bulk table (or [click here](https://miso.gsi.oicr.on.ca/miso/library/bulk/receive?quantity=5&projectId=&sampleClassId=15) if you have navigated away from the page).
+1. Click **Export Data** at the top of the table.
+1. Select **Microsoft Excel** format.
+1. Click **Export**.
+    * The spreadsheet will begin downloading.
+
+<a name="import"/>
+
+### Import
+A spreadsheet can be uploaded to a bulk table page, and any matching data from the spreadsheet will be entered in the table.
+    * In order to match, the spreadsheet column header must exactly match the bulk table column header.
+    * Spreadsheet upload will fail if:
+        * the spreadsheet contains columns whose headers don't match any headers in the bulk table
+        * the spreadsheet contains more rows than the bulk table does
+    * If the spreadsheet cells contain invalid data, the data will still be entered in the table.
+
+1. Open the spreadsheet you downloaded in the [Export](#export) section.
+1. Delete all columns to the right of the _Sci. Name_ column.
+1. Enter text into some or all of the remaining cells.
+1. Save the spreadsheet.
+1. Return to your _Create Libraries_ bulk table (or [click here](https://miso.gsi.oicr.on.ca/miso/library/bulk/receive?quantity=5&projectId=&sampleClassId=15) if you have navigated away from the page).
+1. Click **Import**.
+1. Choose the spreadsheet you edited, and click **Import**.
+    * The cell values will change.
+        * If invalid values are not immediately highlighted as such, you can force validation by clicking **Save**.
+
+## Miscellaneous
+
+<a name="cell-colours"/>
+
+### Cell Colours
+Some of the cells in a table can have different colours, each of which indicates a specific meaning:
+  * Red: cell needs to be filled with valid data. (Empty cell is invalid){% if include.detailed == true %}
+  * Yellow (_Sample/Library Alias_ only): this alias is non-standard, and whatever is entered here will be saved (validation will be skipped). [Click here to see a non-standard alias](https://miso.gsi.oicr.on.ca/miso/sample/bulk/edit?ids=236)
+  * Purple (_Identity Alias_ only): This cell is automatically filled in, but there are other items in the dropdown.
+    * To test this: [Click here](https://miso.gsi.oicr.on.ca/miso/sample/bulk/new?quantity=1&projectId=&sampleClassId=16) and enter **10XT** in the _Project_ column and **1** in the _External Name_ column. 
+      *The _Identity Alias_ cell will be purple, and if you click on the down arrow in the cell, you will see more items in the menu.{% endif %}
```

Comment:
This bullet point is getting mashed into the previous line. I'm guessing again because the first level shouldn't be indented